docs/source/guides/area_detectors.rst

```diff
@@ -0,0 +1,610 @@
+        yield from bps.mv(detector.cam.num_images, 1)
+
+        # Execute count
+        yield from count([detector], num=num, delay=delay)
+
+    def detector_series(detector, num_images, exposure_time):
+        """Collect a series of images."""
+
+        # Configure for series acquisition
+        yield from bps.mv(detector.cam.acquire_time, exposure_time)
+        yield from bps.mv(detector.cam.num_images, num_images)
+        yield from bps.mv(detector.cam.image_mode, "Multiple")
+
+        # Trigger acquisition
+        yield from bps.trigger_and_read([detector])
+
+**Detector Alignment Plans:**
+
+.. code-block:: python
+
+    # plans/detector_alignment.py - Detector positioning
+    from apstools.plans import lineup2
+    from bluesky import plan_stubs as bps
+
+    def align_detector_distance(detector, distance_motor, nominal_distance):
+        """Align detector to optimal distance."""
+
+        # Scan around nominal position
+        yield from lineup2(
+            [detector.stats1.total],
+            distance_motor,
+            nominal_distance - 10,  # mm
+            nominal_distance + 10,  # mm
+            21
+        )
+
+Data Management Integration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Metadata Collection:**
+
+.. code-block:: python
+
+    # devices/detector_metadata.py - Metadata integration
+    from ophyd import Device, Component as Cpt, Signal
+
+    class DetectorMetadata(Device):
+        """Collect detector metadata for data management."""
+
+        # Detector configuration
+        exposure_time = Cpt(Signal, kind="config")
+        num_images = Cpt(Signal, kind="config")
+        detector_distance = Cpt(Signal, kind="config")
+
+        # Environmental conditions
+        detector_temperature = Cpt(EpicsSignal, ":TEMP:RBV", kind="config")
+
+        # Calibration information
+        pixel_size = Cpt(Signal, value=0.172, kind="config")  # mm
+        wavelength = Cpt(Signal, kind="config")  # Angstroms
+
+**File Management:**
+
+.. code-block:: python
+
+    # callbacks/detector_files.py - File management
+    from apstools.callbacks import NXWriter
+    from pathlib import Path
+
+    class DetectorFileManager:
+        """Manage detector files and metadata."""
+
+        def __init__(self, detector, base_path="/data"):
+            self.detector = detector
+            self.base_path = Path(base_path)
+
+        def setup_scan_files(self, scan_id, sample_name):
+            """Configure files for a scan."""
+
+            scan_dir = self.base_path / f"scan_{scan_id:04d}"
+            scan_dir.mkdir(exist_ok=True)
+
+            # Configure HDF5 file
+            self.detector.hdf5.file_path.put(str(scan_dir))
+            self.detector.hdf5.file_name.put(f"{sample_name}")
+
+            # Setup NeXus writer
+            nx_writer = NXWriter(str(scan_dir / f"{sample_name}.nx.hdf5"))
+            return nx_writer
+
+Troubleshooting Area Detectors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Common Issues:**
+
```

Comment:
The content provided here are suggestions for diagnostics.  What are the *common issues* where each of these should be applied?

This section is not helpful in its current form and should be removed.